src/instructlab/model/evaluate.py

```diff
@@ -516,6 +560,14 @@ def validate_options(
         if benchmark == Benchmark.MMLU_BRANCH:
             validate_model(base_model, "--base-model", allow_gguf=False)
 
+    if benchmark == Benchmark.DK_BENCH:
```

Comment:
It seems like at this point we're jamming the functionality of many different evaluations under one command. I don't actually think this is a problem at all, but we should make the validation logic more explicit that something all commands do so that we can have more uniform behavior across them.

Reply:
Yeah it would be great to have a set of common validation functions across all the commands!